server/API.py

```diff
@@ -0,0 +1,10 @@
+import secrets
+import requests
+
+BASE_URL_V3 = 'https://api-v3.mbta.com/{command}?filter[{name}]={value}{suffix}'
+
+def getV3(command, filter_name, filter_value, suffix='', api_key=secrets.API_KEY):
```

Comment:
Why have `suffix` as a separate parameter when it's just getting combined with `value`?

Reply:
It's there so one can request other resources easily; so suffix can be "&include=stop", for example. Maybe it would be more useful as a dictionary that gets interpreted into query parameters? So the function signature could look like this?

`def getV3(command, filter_name, filter_value, params={}, api_key=secrets.API_KEY):`

And you could pass in:
```
{
'include': 'stop'
}
```

Reply:
Yeah, I think a dictionary would be better. You could either provide lots of structure, with separate include and filter params, or little structure and let the caller make their own `filter[x]=y` strings, but being inbetween and providing only a little structure seems like the worst of both worlds. Of the two extremes, I like the less structured dictionary of params better.